pylint_django/transforms/foreignkey.py

```diff
@@ -41,6 +41,17 @@ def _get_model_class_defs_from_module(module, model_name, module_name):
     return class_defs
 
 
+def _module_name_from_django_model_resolution(model_name, module_name):
+    import django  # pylint: disable=import-outside-toplevel
+    django.setup()
+    from django.apps import apps  # pylint: disable=import-outside-toplevel
```

Comment:
Moving the imports into the top-level broke the build during the `django_not_installed` check: https://travis-ci.org/github/PyCQA/pylint-django/builds/710245712 .